go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/go-chi/chi v4.0.2+incompatible
 	github.com/go-ozzo/ozzo-validation v3.6.0+incompatible
 	github.com/go-ozzo/ozzo-validation/v4 v4.3.0
-	github.com/golang/protobuf v1.5.2
+	github.com/golang/protobuf v1.5.2 // indirect
```

Comment:
I wonder what still depends on it.

Reply:
probably github.com/contiamo/goserver

Reply:
hmmmm it is pined to v1.3.5 https://github.com/contiamo/goserver/blob/798094efecde8127a129a0237e8a9ab192d723f0/go.mod#L8